### PR TITLE
MV-247 Storybook configuration

### PR DESCRIPTION
**Pull request by alexmaar**

### CODE REVIEW

**Review by kamil-stasiak — APPROVED** [review]

**Comment by kamil-stasiak** [comment]
Please add a Jira task ID (MV-247) at the begging of PR name

File: assets/.storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```
Comment (pkrucz00) [inline assets/.storybook/preview.js:9]: Please add a new line

**Review by pkrucz00 — APPROVED** [review]
The PR looks good but since the projects requires webpack to be installed in a special way, it would be in good taste to add that line in `readme` (even if it's only a temporary measure).

**Review by bblaszkow06 — APPROVED** [review]
image Just JS things xD

**Pull request closed without merging**